### ISSUE TITLE
[TF2] Halloween 2012 unusual effect "Misty Skull" missing proper particles

### ISSUE THREAD

**Issue by ill5-com** [ValveSoftware/Source-1-Games#4316]
The Halloween 2012 "Spectral Halloween Special" unusual effect "Misty Skull" is missing particle effects the unusual effect is intended to have. 

The official sprite for the unusual effect "Misty Skull" shows the skull having red glowing eyes, however the in-game effect does not have any eyes. If you inspect the unusual effect "Misty Skull" in the particle editor, you can see it is supposed to have the child particle definition "unusual_skull_misty_eyeglow", however it is missing it for unknown reasons. Please see attached pictures.

Official particle effect preview sprite ripped from game files:
![https://scrap.tf/img/particles_440/44_380x380.png](https://scrap.tf/img/particles_440/44_380x380.png)

TF2 Wiki in-game screenshot:
![https://wiki.teamfortress.com/w/images/thumb/b/b0/Unusual_Misty_Skull.png/1024px-Unusual_Misty_Skull.png](https://wiki.teamfortress.com/w/images/thumb/b/b0/Unusual_Misty_Skull.png/1024px-Unusual_Misty_Skull.png)

Demonstration of the missing child particle definition in the particle editor:
[https://www.youtube.com/watch?v=54zD6vjiffM](https://www.youtube.com/watch?v=54zD6vjiffM)

**Comment by matty4kek** [comment]
Don't quote me on that, but I'm fairly sure the skull with red eyes is being used on effect "It's a secret to everybody".

**Comment by ill5-com** [comment]
> Don't quote me on that, but I'm fairly sure the skull with red eyes is being used on effect "It's a secret to everybody".

You are correct, however it is using it's own particle effect with it's own name. I would assume it to be something along the lines of "unusual_skull_mystery_eyeglow" as opposed to "unusual_skull_misty_eyeglow" which is intended for the "Misty Skull" unusual effect, hence the name.

I would tell you exactly which eyeglow particle the "It's a secret to everybody" unusual effect is using and go into more detail but I'm on a mobile device right now. I will update this post with further information when a development environment is available.

**Comment by ill5-com** [comment]
> > Don't quote me on that, but I'm fairly sure the skull with red eyes is being used on effect "It's a secret to everybody".
> 
> You are correct, however it is using it's own particle effect with it's own name. I would assume it to be something along the lines of "unusual_skull_mystery_eyeglow" as opposed to "unusual_skull_misty_eyeglow" which is intended for the "Misty Skull" unusual effect, hence the name.
> 
> I would tell you exactly which eyeglow particle the "It's a secret to everybody" unusual effect is using and go into more detail but I'm on a mobile device right now. I will update this post with further information when a development environment is available.

After inspecting the unusual effect "It's a secret to everybody" which is known as "unusual_fullmoon_cloudy_secret" I can confirm that this is indeed a bug/accidental omission instead of Valve choosing to use the "Misty Skull" eyes (unusual_skull_misty_eyeglow) on "It's a secret to everybody".

As you can see in the screenshot below, the unusual effect "It's a secret to everybody" uses it's own different eyeglow which "blinks" or fades in and out. It is a child particle known as "unusual_fullmoon_cloudy_secret_eyeglow".

![https://i.imgur.com/HqJ3oVR.png](https://i.imgur.com/HqJ3oVR.png)

I would suggest the fix to be to add the missing child particle effect (unusual_skull_misty_eyeglow) to the "Misty Skull" unusual effect. To prevent a disruption in item price for those who liked the effect without eyes, I would suggest adding a "style" to the adjust unusual effect menu that allows the user to decide if they want the glowing eyes or not.
